runtime/manual/basics/modules/private_registries.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Private Registries"
+---
+
+Deno supports private registries, which allow you to host and share your own
+modules. This is useful for organizations that want to keep their code private
+or for individuals who want to share their code with a select group of people.
+
+## What are private registries?
+
+Large organizations often host their own private npm registries to manage
+internal packages securely. These private registries serve as repositories where
+organizations can publish and store their proprietary or custom packages. Unlike
+public npm registries, private registries are accessible only to authorized
+users within the organization.
+
+## How to use private registries with Deno
+
+First, configure your `.npmrc` file to point to your private registry. You can
+do this by adding the following line to your `.npmrc` file:
```

Comment:
Maybe worth linking to npm's own docs here as well?

Reply:
Which ones were you thinking particularly?

Reply:
We could link to https://docs.npmjs.com/cli/v10/configuring-npm/npmrc